tagstudio/src/qt/widgets/media_player.py

```diff
@@ -90,6 +90,14 @@ def __init__(self, driver: "QtDriver") -> None:
 
         self.media_btns_layout.addWidget(self.mute)
 
+        self.volume_slider = QSlider()
+        self.volume_slider.setOrientation(Qt.Orientation.Horizontal)
+        self.volume_slider.valueChanged.connect(self.volume_slider_changed)
+        # set slider value to current volume
+        self.volume_slider.setValue(int(self.player.audioOutput().volume() * 100))
```

Comment:
```suggestion
        # set slider value to current volume
        self.volume_slider.setValue(int(self.player.audioOutput().volume() * 100))
        self.volume_slider.valueChanged.connect(self.volume_slider_changed)
```
or it will set the player's volume to it's current volume (only first time). which isn't necessary.